### ISSUE TITLE
Is prune using start time or end time of archives?

### ISSUE THREAD

**Issue by nesneros** [borgbackup/borg#1143]
In the help for prune it says "The time that each backup completes is used for pruning purposes". Is that really so? It seems to me that it is the start time that is used - not the completion time.

```
$ borg prune --list -v -H 2 /media/jan/OS/borgbackup
Keeping archive: xps15-jan-2016-06-06T12:05:47        Mon, 2016-06-06 14:05:47
Keeping archive: xps15-jan-2016-06-06T11:54:40        Mon, 2016-06-06 13:54:40
```

The archive ' xps15-jan-2016-06-06T11:54:40' took 9 minutees to create and hence completed in the same hour as the other archive. If completion time was used for pruning shouldn't that archive be deleted?

If it is really the complete time that should be used, I also think it would make more sense to display that instead of the start time in the output.

I created the archive with --stats option and got
```
Archive name: xps15-jan-2016-06-06T11:54:40
Archive fingerprint: 9f39548959b628305ad34917cc1d2c772f5135e63068a4b174157653f77f5b0f
Time (start): Mon, 2016-06-06 13:54:41
Time (end):   Mon, 2016-06-06 14:03:48
Duration: 9 minutes 6.96 seconds
Number of files: 100993
```

**Comment by ThomasWaldmann** [comment]
Yes, looks like a documentation bug.

prune uses the start times of the archives to compute what to keep (and what to delete).

**Comment by ThomasWaldmann** [comment]
Fixed by PR #1144 in 1.0-maint, soon to be merged into master also.
